src/executor.rs

```diff
@@ -727,20 +727,20 @@ impl<T> Task<T> {
     /// # Examples
     ///
     /// ```
-    /// use scipio::{LocalExecutor, Timer};
+    /// use scipio::{LocalExecutor, Timer, Task};
     /// use futures_lite::future;
     ///
     /// let ex = LocalExecutor::new(None).expect("failed to create local executor");
     ///
     /// ex.spawn(async {
     ///     loop {
     ///         println!("I'm a background task looping forever.");
-    ///         future::yield_now().await;
```

Comment:
A general comment about your commit message. Doing it here just because github doesn't allow comments on commit messages =)

This API is not unpublished as you call it. It's part of standard rust. The problem is that it is blocking.

I wish there was a way to detect and block automatically usage of this to make sure users are not falling into this trap. If that doesn't prove to be possible then we should update the README.

code is good.